modules/dp_faq/dp_faq.module

```diff
@@ -21,15 +21,3 @@
  * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301,
  * USA.
  */
-
-use Drupal\Core\Form\FormStateInterface;
-
-/**
- * Implements hook_form_FORM_ID_alter().
- */
-function dp_faq_form_views_exposed_form_alter(array &$form, FormStateInterface $form_state, string $form_id): void {
-  if ($form['#id'] === 'views-exposed-form-devportal-faq-page-1') {
-    $form['#attached']['library'][] = 'dp_faq/faq';
-    $form['field_faq_tags_target_id']['#options']['All'] = t('All topics');
-  }
-}
```

Comment:
Please do not leave a file without a single line of working code. This file is not needed for Drupal to recognize the contents of this directory as a module, since it still has an `.info.yml`.